api/backup.go

```diff
@@ -104,11 +104,10 @@ type BackupCreate struct {
 
 	// Domain specific fields
 	Name                    string               `jsonapi:"attr,name"`
-	Status                  BackupStatus         `jsonapi:"attr,status"`
 	Type                    BackupType           `jsonapi:"attr,type"`
 	StorageBackend          BackupStorageBackend `jsonapi:"attr,storageBackend"`
-	MigrationHistoryVersion string               `jsonapi:"attr,migrationHistoryVersion"`
-	Path                    string               `jsonapi:"attr,path"`
+	MigrationHistoryVersion string
```

Comment:
migrationHistoryVersion is used in BackupTable.vue

Reply:
Yeah, I think that is used for Backup instead of this BackupCreate.
https://github.com/bytebase/bytebase/blob/9286757b60f02127630fc8a91b71128911707bf2/frontend/src/components/BackupTable.vue#L262